cpp/oneapi/dal/algo/subgraph_isomorphism/backend/cpu/compiler_adapt.hpp

```diff
@@ -16,7 +17,11 @@
 
 #pragma once
 #include <cstdint>
-#include <immintrin.h>
+
+#ifndef __ARM_ARCH
```

Comment:
Something like `#ifdef TARGET_X86_64` - it shouldn't be included if Arm isn't defined. It should be included if x86 is defined.